src/app/utils/experiments/UnleashProvider.tsx

```diff
@@ -14,9 +14,9 @@ export const UnleashContext = createContext<UnleashContext>({ lastUpdate: null }
 export function UnleashProvider({ children }: { children?: React.ReactNode }) {
   const [lastUpdate, setLastUpdate] = useState<UnleashContext["lastUpdate"]>(null)
   const isHydrated = GlobalStore.useAppState((state) => state.sessionState.isHydrated)
```

Comment:
do we also need to do the same here when the app is rehydrating?
```suggestion
  const isHydrated = GlobalStore.useAppState((state) => state?.sessionState?.isHydrated)
```

Reply:
no this doesn't crash ever for some reason ~ but will monitor and address in case it has the same

Reply:
perfect - thanks